src/App.jsx

```diff
@@ -1,12 +1,45 @@
-import { useState } from 'react'
+import { useEffect, useState } from "react";
 import './App.css'
+import Header from '../src/components/Header'
+import ArticlesList from './components/ArticlesList'
+import { Route, Routes } from "react-router-dom";
+import { getArticles } from "./utils";
+import { getTopics } from "./utils";
+
+
 
 function App() {
+  const [allArticles, setAllArticles] = useState([])
+  const [allTopics, setAllTopics] = useState([])
+
+  useEffect(() => {
+    getArticles().then((response) => {
+      console.log(response)
+      setAllArticles(response);
+    });
+    
+  }, []);
+  useEffect(() => {
+    getTopics().then((response) => {
+      console.log(response)
+      setAllTopics(response);
+    });
+    
+  }, []);
+
+  
 
 
   return (
-    <div>
-      NC News
+    <div className='App'>
+     <Header />
+     <Routes>
+     <Route
+          path="/articles"
```

Comment:
You don't have a route for '/' which is the default for this webpage - please consider if you want someone to have to click a button to access the ArticleList, or whether you want that to be the first page.